interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/bind-to-data/bind-a-report-to-a-federated-master-detail-data-source.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Bind a Report to a Federated Master-Detail Data Source
+owner: Anna Gubareva
+---
+# Bind a Report to a Federated Master-Detail Data Source
+
+This topic describes how to create a federated data source that retrieves data from multiple data sources into separate queries. The topic also shows how to specify a master-detail relationship between these queries.
```

Comment:
"that retrieves data from multiple data sources into separate queries" How can it 'retrieve data' 'into separate queries'?  'Retrieve something into something else' is incorrect.

Reply:
Rewrite